tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<ul><li>Learning goal, first bullet: from a ^spectral^ cube.</li><li>keywords to add: spectral cube, pvextractor, radio astronomy, coordinates</li><li>Summary: cube -&gt; spectral cube.</li><li>Rest of summary is excellent!</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done